doc/deploying/controlling-with-url-parameters.md

```diff
@@ -20,7 +21,7 @@ Parameter      | Meaning
 `mode=preview`   | Operate in "preview mode", which mostly means not showing a warning if the screen is `small`  
 `share=`...      | Load a map view previously saved using the "Share" function with URL shortening.
 `start=`...      | Load a map view previously saved without URL shortening. The argument is a URL-encoded JSON structure defined using an internal format described below.
-`<initfile>`     | Load catalog file as described below.
+`<initfile>`     | Load [init/catalog file](../customizing/initialization-files.md) as described below.
```

Comment:
Maybe need to reiterate this bit here for the people who skim read:
NB: You do not need to specify <initfile> in the url, as any unrecognised parameter (eg `foo`) is treated as the name of a catalog file